cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -695,8 +695,7 @@ def test_advanced_components_in_edit_unit(self):
         # page response HTML
         self.check_components_on_page(
             ADVANCED_COMPONENT_TYPES,
-            ['Word cloud', 'Annotation', 'Text Annotation', 'Video Annotation', 'Image Annotation',
-             'split_test'],
+            ['Word cloud', 'Annotation', 'split_test'],
```

Comment:
Wait, actually why is 'Annotation' still on this list?

Reply:
"Annotation" is not a component I built and has nothing to do with the code base that is used in Text Annotation, Video Annotation, or Image Annotation.

Reply:
Wow... I had long assumed that they were related. But okay, great. Annotation was the one with Studio docs, so we don't have to worry about doc updates then I guess.

Thank you!